### ISSUE TITLE
yarpidl_rosmsg generates an invalid file for std_msgs/Empty.msg

### ISSUE THREAD

**Issue by drdanz** [robotology/yarp#1566]
The file generated by yarpidl_rosmsg for the `Empty` message from the `std_msgs` package is broken.

**Comment by drdanz** [comment]
The generated file contains something like this:

```c++
class std_msgs_Empty : public yarp::os::idl::WirePortable
{
public:

    std_msgs_Empty()    }
```

**Comment by Nicogene** [comment]
Closed by #1572
